NOTICE

```diff
@@ -1,594 +1,1045 @@
-#####################################################################################BEGIN CLOUDERA DIRECTOR NOTICES
-Items appearing in the following notices are provided merely for the notification and guidance of the recipient in accordance with the listed license.  In an abundance of caution, the listing includes various dependencies applicable only to development, testing, and evaluation, which may not be included in the distribution and, indeed, may have been deprecated in favor of alternatives.  Accordingly, this listing should not be construed as an admission that the listed component appears in the distribution or in the manner indicated.  Similarly, fulfilling a license's compliance requirements for a particular form of distribution is not an admission that the software is in fact distributed in that form.  Distributions and packaging vary and not all third party components are relied upon in the same manner or degree.  In addition, some compliance documentation, such as embedded notices files in source code, may be found within the distribution itself rather than below.
+#####################################################################################BEGIN CLOUDERA ALTUS DIRECTOR NOTICES
+Items appearing in the following notices.txt are provided for the notification and guidance of the recipient in accordance with the listed license.  In an abundance of caution, the listing may include various dependencies applicable only to development, testing, and evaluation, which may not be included in the distribution and, indeed, may have been deprecated in favor of alternatives.  Accordingly, this listing should not be construed as an admission that the listed component appears in the distribution or in the manner indicated.  Similarly, fulfilling a license’s compliance requirements for a particular form of distribution is not an admission that the software is in fact distributed in that form.  Distributions and packaging vary and not all third party components are relied upon in the same manner or degree.  In addition, some compliance documentation, such as embedded notices files in source code, may be found within the distribution itself rather than below.
 
 APACHE 2.0 LICENSED THIRD PARTY CODE
 -------------------------------------------------------------------
 
-AssertJ Core 1.6.0
```

Comment:
I presume the rest of this file was automatically generated rather than manually updated. Can you share the command you ran to generate this set of notice files in a file somewhere in this repo so that it's clear how to regenerate it in the future?

I presume it was run from just this specific open-source repo and not reliant on any non-open-source components, so it should be possible to share it and reproduce this change verbatim.

Otherwise, we're just rubber-stamping a huge diff without ability to review it.

Reply:
It is automatically generated. Like I said, it's given to us by our legal department; the engineering team doesn't make it. I treat it as a black box. I completely agree that it is not reviewable due to its size and the quantity of changes in it.

We only get one notice for all things Director; it was decided it was too much work (for our legal folks) to generate separate notices for Director's core and every plugin. So this document covers more than just this plugin here.

Ping me offline if you want to discuss this stuff with our legal department and I'll get you in touch with them.

Reply:
Ack. No need to start a thread on this, I trust they know what they're doing. Thanks for the clarification!

Reply:
No problem, thanks for your flexibility!